browser_patches/firefox/juggler/protocol/Protocol.js

```diff
@@ -2,13 +2,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-const {t, checkScheme} = ChromeUtils.import('chrome://juggler/content/protocol/PrimitiveTypes.js');
+const { t, checkScheme } = ChromeUtils.import(
```

Comment:
looks like a lot of unrelated formatting changes did happen here, can we only include actual impl changes?